gdsfactory/typings.py

```diff
@@ -261,5 +259,6 @@ class Array(np.ndarray[Any, np.dtype[Any]], metaclass=ArrayMeta):
     "Size",
     "Spacing",
     "Strs",
+    "Transition",
```

Comment:
Do we need to explicitly export this? It seems good practice to keep the typings module exports just for types created in this file

Reply:
I can send a separate PR for that